package-lock.json

```diff
@@ -37761,18 +37761,17 @@
 		},
 		"wpcom": {
 			"version": "file:packages/wpcom.js",
-			"integrity": "sha512-fVrJU9U979m9T0B0IEeAfpIQRM397J5NcsWLPcJov87jATubxVt6akZATczAh9rN40H1h2HCj/YtAW6L1zh8QA==",
 			"requires": {
-				"babel-runtime": "^6.9.2",
-				"debug": "^3.1.0",
+				"@babel/runtime": "^7.8.3",
```

Comment:
iiiiinteresting

Reply:
I think that's caused by the recent `wpcom` PRs that upgraded the package's dependencies, but not the lockfile.

Reply:
yeah, surprised `npm ci` wasn't yelling about it

Reply:
I think that `npm ci` doesn't look at the `requires` fields at all. It just blindly recreates the `node_modules` tree from the nested `dependencies` objects in the lockfile. For that, only the tree structure and the `resolved` and `integrity` fields are needed.